sdk/digitaltwins/azure-digitaltwins-core/API design.md

```diff
@@ -1178,18 +1178,10 @@ Async APIs
     /**
      * Creates one or many models.
      * @param models The list of models to create. Each string corresponds to exactly one model.
-     * @return The list of created models
+     * @return A {@link PagedFlux} of created models.
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
-    public Mono<List<ModelData>> createModels(List<String> models) { }
-
-    /**
-     * Creates one or many models.
-     * @param models The list of models to create. Each string corresponds to exactly one model.
-     * @return A REST response containing the list of created models.
-     */
-    @ServiceMethod(returns = ReturnType.COLLECTION)
-    public Mono<Response<List<ModelData>>> createModelsWithResponse(List<String> models) { }
+    public PagedFlux<ModelData> createModels(Iterable<String> models) { }
```

Comment:
If the PL accepts a `List`, then there isn't much value in us accepting an `Iterable`.